gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/replication/ConfigBasedDatasetsFinder.java

```diff
@@ -265,19 +268,21 @@ public Path commonDatasetRoot() {
     if (leafDatasets.isEmpty()) {
       return ImmutableList.of();
     }
+    log.debug(String.format("[ConfigBasedDatasetsFinder] Find %d in the commonRoot:%s", leafDatasets.size(), this.commonRoot));
 
     // Parallel execution for copyDataset for performance consideration.
     final List<Dataset> result = new CopyOnWriteArrayList<>();
     Iterator<Callable<Void>> callableIterator =
         Iterators.transform(leafDatasets.iterator(), new Function<URI, Callable<Void>>() {
           @Override
           public Callable<Void> apply(final URI datasetURI) {
-            return findDatasetsCallable(configClient, datasetURI, props, blacklistPatterns, result);
+            log.debug("[ConfigBasedDatasetsFinder] Try to search ConfigBasedDatasets for datasetURI:" + datasetURI);
```

Comment:
Reword the log message to:
"log.debug("..... Searching ConfigBasedDatasets for ....")"